### PR TITLE
Add desktop version of my Portfolio Website

### PR DESCRIPTION
**Pull request by Efiamotu-1**
## Features Added

> - All desktop version images from Figma.
> - Css grid and Flexbox for responsiveness
> - Media Queries for specififying styling for desktop version
> - Use of display none to hide hamburger icon on desktop view and display block to show it on mobile view
> - nav responsive to both desktop and mobile view.
> - update readme file to add names of coding partners who contributed to my code

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -50,133 +62,121 @@ <h3>I’am a Software Developer</h3>
...
+        </p>
+        <ul class="project-techs">
           <li>html</li>
           <li>bootstrap</li>
           <li>Ruby</li>
         </ul>
-        <a href="#">
-          See Project
-        </a>
+
```
Comment (fobadara) [inline index.html:110]: [OPTIONAL] Please ensure that you stick to the Figma design as much as possible. This tells recruiters that you have an eye for details. You can check the image below for some examples:

| Your Design | Figma Design |
|-------------|---------------|
|Figma_cards_habeeb|Figma_card|

This also includes other cards

File: index.html [hunk 2]
```diff
@@ -195,12 +195,12 @@ <h1>Language</h1>
     </ul>
     </div>
```
Comment (fobadara) [inline index.html:197]: - [ ] Pease kindly align your list for languages, frameworks and skills to the left

| Your Design | Figma Design |
|-------------|---------------|
|||

File: index.html [hunk 3]
```diff
@@ -195,12 +195,12 @@ <h1>Language</h1>
...
+    <div class="lang">
+      <div class="lang-heading">
         <img src="images/shapes/Rectangle-56.png" alt="">
         <h1>Frameworks</h1>
       </div>
-      <ul>
+      <ul class="lang-skills">
       <li>
         Bootstrap
       </li>
```
Comment (fobadara) [inline index.html:206]: - [ ] kindly align to the left as stated above

File: index.html [hunk 4]
```diff
@@ -253,19 +253,20 @@ <h1>Skills</h1>
   </div>
```
Comment (fobadara) [inline index.html:253]: - [ ] Kindly align to the left as explained above

File: index.html [hunk 5]
```diff
@@ -253,19 +253,20 @@ <h1>Skills</h1>
...
       <li>
-        <input type="email" id="email" name="email" placeholder="Email Address" required>
+        <input class="form-list" type="email" id="email" name="email" placeholder="Email Address" required>
       </li>
       <li>
-        <textarea name="message" id="textarea" maxlength="500" required>Enter text here</textarea>
+        <textarea class="form-textarea" name="message" id="textarea" maxlength="500" required>Enter text here</textarea>
       </li>
       <li class="button-get-in-touch">
         <button type="submit" class="btn">Get in touch</button>
```
Comment (fobadara) [inline index.html]: [OPTIONAL] You don't need to wrap your inputs in `ul` and `li` tags. They can stand alone.

File: index.html [hunk 6]
```diff
@@ -292,6 +293,7 @@ <h1>I'm always interested in hearing about new projects, so if you'd like to cha
       <a href="#"> <img src="images/icons/medium.png" alt=""> </a>
     </li>
    </ul>
```
Comment (fobadara) [inline index.html:289]: - [ ] Kindly move your social links a little to the right as shown in the template

| Your Design | Figma Design |
|-------------|---------------|
|Figma_social_habeeb|Figma_social|

File: styles.css [hunk 7]
```diff
@@ -1,70 +1,25 @@
...
 @import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:ital,wght@0,100;0,200;1,100&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
-@import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@100;200;300;400;500;600&family=Poppins:ital,wght@0,100;0,200;1,100&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
 
-* {
-  margin: 0;
-  padding: 0%;
-  box-sizing: border-box;
-  text-decoration: none;
-  list-style-type: none;
-}
```
Comment (fobadara) [inline styles.css:12]: [OPTIONAL] kindly check out [this link](https://frontend.turing.edu/lessons/module-1/reset-vs-normalize.html) on Resetters and Normalizers. I suggest you get familiar with Normalizers.

File: styles.css [hunk 8]
```diff
@@ -1,70 +1,25 @@
...
-  padding-top: 20px;
-}
-      
-.logo a{
-  color: white;
-  margin-left: 20px;
-  font-weight: 700;
-  font-size: 18px;
-  line-height: 24px;
-}
```
Comment (fobadara) [inline styles.css:39]: - [ ] Kindly follow [this link](https://www.freecodecamp.org/news/taking-the-right-approach-to-responsive-web-design/#:~:text=What%20is%20mobile%2Dfirst,styling%20for%20larger%20screen%20sizes.&text=In%20that%20example%2C%20for%20small,re%20simply%20applying%20some%20padding.) on mobile-first design. That is probably why your site is overflowing at a width of 375px. You should be using min-width not max-width.

| You Deign |
|-----------|
|Figma_overflow|

File: README.md [hunk 9]
```diff
@@ -13,25 +13,30 @@ This is my portfolio website which shows my various projects and showcases my sk
...
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+## Authors
 
+👤 **Habeeb Musa Efiamotu**
 
+- GitHub: [@githubhandle](https://github.com/Efiamotu-1)
+- Twitter: [@twitterhandle](https://twitter.com/EFYAMOTU)
+- LinkedIn: [LinkedIn](www.linkedin.com/in/musa-habeeb)
```
Comment (fobadara) [inline README.md]: - [ ] This link is not working. Please fix it

**Review by fobadara — CHANGES_REQUESTED** [review]
# REQUIRED CHANGES ♻️ 

Hi @Efiamotu-1, 

Good job so far! 👍🏽 👍🏽 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💡 
✅ Linters are all green 👍🏽 
✅  Form submits

## Required Changes ♻️
- I noticed that you used a lot of `h1` tags. This opposes HTML best practices and is not good for SEO and also could mess with screen readers therefore affecting accessibility. You should use the `h1` tag only once. For example in a post about a library the library name would be in `h1`, science books in `h2`, maths books in another `h2`, biology books in an `h3`, books on algebra in another `h3` and son on. Please read up on [heading tags](https://www.contentkingapp.com/academy/headings/#:~:text=robots.txt%20guide%3A-,Heading%20hierarchy,even%20add%20H4%2DH6%20subheadings%20.).
- Your mobile version now overflows at a width of 375px. Please make sure it doesn't at this width upwards. I think you should look into Responsive Web Design  and Mobile-first approach.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider:
- [OPTIONAL] Kindly commit more often at logical points. For example when you are done with HTML if it is not too large or when you are done styling some parts of your page. Good job 👍🏽  making your commit message in the imperative but you should also take note of the case
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 8 above)
Comment (Efiamotu-1) [reply styles.css:39]: this is my portfolio on screen size of 375px, it is perfectly fine, i am totally unaware of why it showed that way when you checked

Comment (Efiamotu-1) [reply styles.css:39]: @fobadara

File: styles.css (hunk 7 above)
Comment (Efiamotu-1) [reply styles.css:12]: i will checkout this link to learn more about this, however styles.css isnt connected to my html page you can check to confirm that,i just wanted to confirm that. @fobadara thank you

File: index.html (hunk 5 above)
Comment (Efiamotu-1) [reply index.html]: thank you for this as well, however i will like to learn more about when to use ul to display items i would be grateful if you can share a link. thank you @fobadara

**Review by Ugboaja-Uchechi — APPROVED** [review]
Hi @Efiamotu-1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Linters are passing ✔️ 
- Great job with your design 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**